x/gov/types/params.go

```diff
@@ -17,7 +17,7 @@ const (
 
 // Default governance params
 var (
-	DefaultMinDepositTokens = sdk.TokensFromConsensusPower(10)
+	DefaultMinDepositTokens = sdk.TokensFromConsensusPower(10, sdk.DefaultPowerReduction)
```

Comment:
This should just be 1 right @sunnya97? We don't want this denominated in ConsensusPower long term. (Perhaps an issue for a different PR though)

Reply:
Yeah.  Don't know why governance min deposits need to be denominated in power reduction.  @psaradev can you make a new issue to switch this in a future PR?

Reply:
@sunnya97 what do I have to do here to determine min deposit tokens? It's already deterministic by governance using on-chain param feature or manually by code.

Reply:
It's not cause of determinism, it's more just that it's not really related.  There's no reason the default min deposit needs to be based on PowerReduction, which is more something that's internal to `staking`.

I think for the default, we can just set this to `sdk.NewInt(10)` or something.  Or even `sdk.NewInt(10000000)` which would be the same as what it is now, without the dependency.